<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>435</width>
    <height>324</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QCheckBox" name="stepperToggle">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="stepperDirection">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>40</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Direction</string>
   </property>
  </widget>
  <widget class="QSlider" name="servoSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1000</number>
   </property>
   <property name="maximum">
    <number>2000</number>
   </property>
   <property name="singleStep">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="stepperSlider">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>160</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimum">
    <number>4</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>4</number>
   </property>
   <property name="value">
    <number>4</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="blueToggle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue LED</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="greenToggle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>170</y>
     <width>72</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>Green LED</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lcdLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="maxLength">
    <number>16</number>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>130</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Servo Motor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Stepper Motor</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Speed</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>20</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ultrasonic Sensor</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sensorToggle">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>61</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>On/Off</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="sensorBar">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>80</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="maximum">
    <number>5000</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="format">
    <string>%pmm</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>LCD</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
